R/release.R

```diff
@@ -446,7 +446,7 @@ is_rstudio_person_canonical <- function() {
     "fnd" %in% roles &&
     "cph" %in% roles &&
     attr(roles, "appears_in", exact = TRUE) == "given" &&
-    attr(roles, "appears_as", exact = TRUE) == "RStudio"
+    attr(roles, "appears_as", exact = TRUE) == "Posit, PBC"
```

Comment:
Could we update the function name here?

Reply:
Yes, definitely for this one... I wasn't sure how far down that hole to go in this PR (e.g., `is_rstudio_pkg`, `is_rstudio_cph_or_fnd`, `get_rstudio_roles`) - though I'm happy to if you'd like!

Reply:
I can live with doing a bit of this function renaming in this PR. Or doing none of it now and making a separate PR. Or doing all of it. It's an internal matter.

Reply:
I've done most of this renaming in 5b954cec9. I added the small helper function `author_has_rstudio_email()` which will also apply to #1683.